<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffTypeTemplates</class>
 <widget class="QWidget" name="StaffTypeTemplates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="staffTypeSelector">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>391</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="newStaffTypeButton">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadStaffTypeButton">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeStaffTypeButton">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="duplicateStaffTypeButton">
      <property name="text">
       <string>Duplicate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="saveStaffTypesButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
